LICENSES/LicenseRef-NVIDIA-CG.txt

```diff
@@ -0,0 +1,138 @@
+License For Customer Use of NVIDIA Software
```

Comment:
*About filename:* Is there some standard which prefixes the filename with `LicenseRef-`? I'd have expected it in the SPDX tag, but not in the filename itself.

Reply:
https://reuse.software/faq/#custom-license mentions this. I find it a bit awkward as well, I did it incorrectly at first and had `reuse lint` complain about it.